iot-hub/Samples/service/ImportExportDevicesSample/ImportExportDevicesSample.cs

```diff
@@ -44,37 +46,36 @@ public class ImportExportDevicesSample
         }
 
         public async Task RunSampleAsync(
-            bool addDevices,
-            int numToAdd,
-            bool copyDevices,
-            bool deleteSourceDevices,
-            bool deleteDestDevices)
+            int devicesToAdd,
+            bool shouldCopyDevices,
+            bool shouldDeleteSourceDevices,
+            bool shouldDeleteDestDevices)
         {
             // This sets cloud blob container and returns container uri (w/shared access token).
             _containerUriWithSas = await PrepareStorageForImportExportAsync(_storageAccountConnectionString).ConfigureAwait(false);
 
-            if (addDevices)
+            if (devicesToAdd > 0)
```

Comment:
We don't need the `addDevices` bool parameter if we can simply use the number of devices to add being greater than 0 to decide whether or not to do it.